src/pages/cli/graphql/authorization-rules.mdx

```diff
@@ -99,6 +99,8 @@ By default, the Cognito user pool's user information is populated into the `owne
 
 You can override the `owner` field to your own preferred field, by specifying a custom `ownerField` in the authorization rule.
 
+It is not recommended to set your `ownerField` to be the same as your `@primaryKey` field, or `id` if no primary key is specified. If you wish to be able to query by the `ownerField` specified, using an `@index` on that `ownerField` is recommended instead.
```

Comment:
It'd be great if this had a bit more context on why—it's not quite clear to me why this isn't a valid pattern, per https://github.com/aws-amplify/amplify-category-api/issues/872#issuecomment-1278488889

Reply:
@alharris-at - Wanted to bump this to see if the conversation can be resolved. Thank you!